Dockerfiles/mariadb/mariadb.conf

```diff
@@ -0,0 +1,15 @@
+[mariadb]
+# File Key Management
+plugin_load_add = file_key_management
+file_key_management_filename = /etc/mysql/encrypt/keys.enc
+file_key_management_filekey = FILE:/etc/mysql/encrypt/pass.key
+file_key_management_encryption_algorithm = AES_CTR
+
+# InnoDB Encryption Setup
+innodb_encrypt_tables = FORCE
+innodb_encrypt_temporary_tables = ON
+innodb_encrypt_log = ON
+innodb_encryption_threads = 4
+
```

Comment:
Do you know and understand the following one?
[https://mariadb.com/kb/en/innodb-system-variables/#innodb_encryption_rotate_key_age]
It is given as `innodb_encryption_rotate_key_age = 1` in basic configuration
[https://mariadb.com/kb/en/innodb-encryption-overview/]
This is confusing as file_key_management plugin doesn't support key rotation. Does this mean that even if the plugin doesn't do it, innodb engine can do it???
What do you think?

Reply:
Innodb supports it, but the file key plugin does not. E.g. the AWS plugin does support key rotation.

Reply:
ok now that I look at the AWS plugin, it makes sense.